components/raftstore-v2/src/router/message.rs

```diff
@@ -244,6 +245,7 @@ pub enum PeerMsg<EK: KvEngine> {
     ApplyRes {
         res: ApplyTaskRes<EK::Snapshot>,
     },
+    FetchedLogs(FetchedLogs),
```

Comment:
SignificantMsg and CasualMsg are obsolete in v2?

Reply:
No, they will be added back by demand.